geosnap/analyze/network.py

```diff
@@ -64,6 +89,17 @@ def isochrones_from_id(origin, network, threshold):
         A single or list of threshold distances for which isochrones will be
         computed. These are in the
         same units as edges from the pandana.Network.edge_df
+    algorithm : str, {'alpha', 'hull'}
```

Comment:
what about for the args: {'alpha' 'concave'} ?

Reply:
I second this.

Reply:
thats probably better than the existing, to avoid people thinking they are getting a *convex* hull, but an alpha shape is still a particular instance of a concave hull, so its not a perfect distinction there either

Reply:
How about renaming `algorithm` to `concave` with acceptable values being `'libpysal'` or `'shapely'` with a bit more explanation in docstring? That may be too involved though?

Reply:
i like that, but i think i prefer hull to concave. Updated in the latest commit